File: pret/DTO/AmortissementDto.cs
using System;

namespace Pret.DTO
{
    public class AmortissementDto
    {
        public int IdAmortissement { get; set; }
        public int IdCompte { get; set; }

        public int Mois { get; set; }
        public decimal Mensualite { get; set; }
        public decimal Interet { get; set; }
        public decimal Capital { get; set; }
        public decimal ResteDu { get; set; }
        public string Statut { get; set; }

        public int CompteIdClient { get; set; }
        public string CompteClientNom { get; set; }
        public string CompteClientPrenom { get; set; }
    }
}

File: server-ejb/src/main/java/com/example/dto/AmortissementDto.java
package com.example.dto;

import lombok.Data;

import java.math.BigDecimal;
import java.time.LocalDate;

@Data
public class AmortissementDto {
    private int idAmortissement;
    private int idCompte;
    private int mois;
    private BigDecimal mensualite;
    private BigDecimal interet;
    private BigDecimal capital;
    private BigDecimal resteDu;
    private LocalDate createdAt;
    private String statut;
    private int compteIdClient;
}
